Sources/Neon/TextViewSystemInterface.swift

```diff
@@ -155,9 +155,9 @@ extension LayoutManagerSystemInterface: TextSystemInterface {
 }
 #endif
 
-/// A concrete `TextSystemInterface` that uses `NSTextLayoutManager` rendering attributes.
+/// A concrete ``TextSystemInterface`` that uses `NSTextLayoutManager` rendering attributes.
 @available(macOS 12.0, iOS 16.0, tvOS 16.0, *)
-public struct `TextLayoutManagerSystemInterface` {
```

Comment:
I believe the backticks were a double-click-to-select copy&paste accident :)

Reply:
wow must have been yes...